app/views/cases/offender_sar/_request_details.html.slim

```diff
@@ -0,0 +1,13 @@
+tr.request_dated.section
+  th
+    = t('common.case.request_dated')
+  td
+    = l case_details.request_dated, format: :default
```

Comment:
There's a crash on this partial that Alex reported if the optional "request_dated" date field is nil. Please could you amend this line to:

```suggestion
    = l(case_details.request_dated, format: :default) if case_details.request_dated.present?
```

Reply:
Done!